src/impls.rs

```diff
@@ -33,6 +33,7 @@ impl_native!(Blob, "Blob");
 impl_native!(bool, "boolean");
 impl_native!(String, "string");
 impl_native!(str, "string");
+impl_native!(&'static str, "string");
```

Comment:
Given that there's already a `impl_native!(str, "string");` and a `impl<T: TypeDef> TypeDef for &'static T` in this file, I would expect this not to be needed. Did you find a situation that required this new impl?

Reply:
Yup, not needed & removed. Thanks!